### PR TITLE
Make the script useable for PbPb

### PR DESCRIPTION
**Pull request by chiarazampolli**
@shahor02 , @catalinristea : please check in detail that I did not forget anything. Thanks!

### CODE REVIEW

File: DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh [hunk 1]
```diff
@@ -100,7 +118,12 @@ DEBRIS=""
 if [[ $BEAMTYPE == "pp" ]]; then
   DEBRIS="pvertexer.maxChi2TZDebris=10"
```
Comment (shahor02) [inline DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh:119]: actually, the `pvertexer.maxChi2TZDebris` is already set to 10 / 2000 for pp / pbpb in the dpl-workflow.sh

**Review by shahor02 — COMMENTED** [review]
Hi @chiarazampolli 
see below. Note that we still have no special treatment of ZDC BC 0, 3563, will do this now.

File: DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh [hunk 2]
```diff
@@ -45,7 +46,20 @@ if [[ $RUNNUMBER -ge 521889 ]]; then
...
+# The "wrong" +1 offset request for ITS (0) must produce alarm since shifts are not supported there
+if [[ $PERIOD == "LHC22s"]]; then
+  if [[ $RUNNUMBER -eq 529403 ]]; then
+    ZDC_BC_SHIFT=213268844053
+  elif [[ $RUNNUMBER -eq 529418 ]]; then
+    ZDC_BC_SHIFT=102488091157
+  else
+    ZDC_BC_SHIFT=0
+  fi
+  export CONFIG_EXTRA_PROCESS_o2_ctf_reader_workflow="TriggerOffsetsParam.customOffset[2]=1;TriggerOffsetsParam.customOffset[4]=1;TriggerOffsetsParam.customOffset[5]=1;TriggerOffsetsParam.customOffset[6]=1;TriggerOffsetsParam.customOffset[7]=1;TriggerOffsetsParam.customOffset[11]=$ZDC_BC_SHIFT;TriggerOffsetsParam.customOffset[0]=1"
```
[the remark this answers is not in the document]
Comment (chiarazampolli) [reply DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh]: For `;TriggerOffsetsParam.customOffset[0]=1`, yes, sorry, I guess it slipped it. 
For the other detectors: please, I will ping them also via mail, to start the apass3.

**Comment by chiarazampolli** [comment]
Hello @catalinristea , 
I also added explicitly setting `pvertexer.timeMarginVertexTime=1.3` for 22c, d, e, f, while we finalize the PR for the IR-depending settings (https://github.com/AliceO2Group/O2DPG/pull/743).

**Comment by catalinristea** [comment]
Ciao Chiara,
Regarding the processing, all you did is fine with me. Thanks!
Yours, Catalin.

File: DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (chiarazampolli) [reply DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh]: from the mail exchange, all seems fine for EMC, PHS, CPV.

**Comment by chiarazampolli** [comment]
Hello @shahor02 , @mconcas,

anything to be added for PbPb for ITS reco? See jira, where Matteo mentioned to possibly adjust further some settings.

chiara

**Comment by mconcas** [comment]
> Hello @shahor02 , @mconcas,
> 
> anything to be added for PbPb for ITS reco? See jira, where Matteo mentioned to possibly adjust further some settings.
> 
> chiara

Hello,
we can try with what we had in old MC studies that is `ITSVertexerParam.clusterContributorsCut=16`.
For better settings I'll need some more time.

**Comment by chiarazampolli** [comment]
Merging, to start apass3.

**Pull request merged**